optashift-employee-rostering-server/src/main/resources/org/optaplanner/openshift/employeerostering/server/solver/employeeRosteringScoreRules.drl

```diff
@@ -118,3 +118,13 @@ rule "Desired time slot for an employee"
     then
         scoreHolder.addSoftConstraintMatch(kcontext, +$tenantConfiguration.getDesiredTimeSlotWeight());
 end
+
+rule "Employee is rotation employee"
+    when
+    	$tenantConfiguration : TenantConfiguration(rotationEmployeeMatchWeight != 0)
+        Shift(  
+                $e: rotationEmployee,
+                employee == $e)
```

Comment:
Instead of rewarding if it's the same, I 'd punish if it's different. So != instead of == and -weight instead of +weight.

Reply:
The variable $e is ambiguous: I'd expect it to represent the employee, not the rotationEmployee.
In any case, we can remove it, simply inline it.

Reply:
If rotationEmployee == null, then this rule shouldn't apply (that's probably the reason why you went with a positive constraint). I still think it needs to be a negative constraint, from my experience, because otherwise the score will look funny (and harder to explain to users - a score of 0 might not be perfect).

Reply:
rule "Desired time slot for an employee" is also a positive constraint, should I change it to a negative (since a score of 0 might not be perfect)?